src/explore-education-statistics-frontend/src/modules/find-statistics/PublicationReleasePage.tsx

```diff
@@ -299,9 +299,14 @@ class PublicationReleasePage extends Component<Props> {
                       )}
                       {data.publication.externalMethodology && (
                         <li>
-                          <Link to={data.publication.externalMethodology.url}>
+                          <a
+                            className="govuk=link"
```

Comment:
Don't need to add a link class (it's also the wrong one anyway)

Reply:
Thanks, I've removed the link class now @ntsim.